terraform-modules/egress_security_group/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_security_group" "security_group" {
+  name        = "${var.name}"
+  description = "${var.description}"
+  vpc_id      = "${var.vpc_id}"
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags {
+    Name = "${var.name}"
+  }
+}
+
+data "aws_vpc_endpoint_service" "logs" {
+  service = "logs"
+}
+
```

Comment:
Is this is the right file, `egress_security_group`?

Reply:
I’m not convinced it’s the best file long-term, but it was where I put it to test with the catalogue pipeline. (It can't go with the VPC because you need the security group IDs.)